<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetaSDFX</class>
 <widget class="QWidget" name="MetaSDFX">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MetaSDF-X</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="openSDFButton">
       <property name="minimumSize">
        <size>
         <width>132</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>132</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Open SDF</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="openMetabocardButton">
       <property name="minimumSize">
        <size>
         <width>132</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>132</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Open Metabocard</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="dbtype">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>32</height>
        </size>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>HMDB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>KEGG</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>DB fields</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QListView" name="listView_2">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Molecules</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0" colspan="3">
         <widget class="QListView" name="listView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QPushButton" name="invertButton">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>133</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Invert </string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>41</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="selectAllButton">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>133</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Select all</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QSpinBox" name="selmol">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>24</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="maximum">
             <number>99999999</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="totalmol">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>24</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="maximum">
             <number>99999999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="selectbylistButton">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>133</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Select by list</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QPushButton" name="unselectButton">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>133</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Unselect</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="showmolname">
          <property name="text">
           <string>Show molecule name</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="saveSDFButton">
       <property name="minimumSize">
        <size>
         <width>132</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>132</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Save SDF</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="extractButton">
       <property name="minimumSize">
        <size>
         <width>132</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>132</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Extract</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>914</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="aboutButton">
       <property name="minimumSize">
        <size>
         <width>132</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>132</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="minimumSize">
        <size>
         <width>132</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>132</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox</zorder>
  <zorder>progressBar</zorder>
  <zorder>aboutButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
